### PR TITLE
Laravel Readme Update

### PR DESCRIPTION
**Pull request by Thiyagu55**
README should have only installation and usage. So moving these sections to a separate file

### CODE REVIEW

**Comment by sjs994** [comment]
Also please provide a description of what change we did and why we did this change in this PR.

**Comment by Thiyagu55** [comment]
> Also please provide a description of what change we did and why we did this change in this PR.

Updated

**Comment by Thiyagu55** [comment]
> Looks good to me.
> 
> Can we have a dev-documentation (either in the same file as a Developer section or a different document) where there will be steps on how to build source code, how to run unit/integration tests and how to test changes ?
> 
> This can be in a separate PR.

Updated

File: php/sqlcommenter-php/packages/sqlcommenter-laravel/README.md [hunk 1]
```diff
@@ -38,9 +43,22 @@ Add the following class above `Illuminate\Database\DatabaseServiceProvider::clas
...
+| `controller`  | <div style="text-align: center">&#10004;</div> | The [name](https://laravel.com/docs/9.x/controllers) of the matching URL pattern as described in your routes/api.php        |
+| `db_driver`   | <div style="text-align: center">&#10004;</div> | The name of the php [database engine](https://laravel.com/docs/9.x/database)                                                |
+| `framework`   | <div style="text-align: center">&#10004;</div> | The word "laravel" and the version of laravel being used                                                                    |
+| `route`       | <div style="text-align: center">&#10004;</div> | The [route](https://laravel.com/docs/9.x/routing) of the matching URL pattern as described in your routes/api.php           |
+| `traceparent` | <div style="text-align: center">&#10004;</div> | The [W3C TraceContext.Traceparent field](https://www.w3.org/TR/trace-context/#traceparent-field) of the OpenTelemetry trace |
+
+## Run Unittests
+In the source repo, run unit tests using below command
```
Comment (sjs994) [inline php/sqlcommenter-php/packages/sqlcommenter-laravel/README.md:61]: Lets either remove this unit test from this PR and checkin that or you have to create a complete "Development" section, which contains information on from where to pull the code (sqlcommenter monorepo), how to build the package, unit test, integration test, integrating the local library with PHP server.

Comment (Thiyagu55) [reply php/sqlcommenter-php/packages/sqlcommenter-laravel/README.md]: Updated

**Review by sjs994 — APPROVED** [review]

**Pull request merged**